deployment/ccip/changeset/solana/cs_chain_contracts.go

```diff
@@ -1,21 +1,30 @@
 package changeset_solana
```

Comment:
Actually a question on internal/solana_contracts. Have we evaluated the possibility of these contracts being required by other packages? Wondering if we should place it under core like we do for gethwrappers?